scripts/deploy-test-pods.sh

```diff
@@ -6,7 +6,16 @@ source "$SCRIPT_DIR"/init-env.sh
 
 mkdir -p ./temp
 cat ./test-target/local-pod-under-test.yaml | APP="testdp" RESOURCE_TYPE="Deployment" MULTUS_ANNOTATION=$MULTUS_ANNOTATION "$SCRIPT_DIR"/mo > ./temp/rendered-local-pod-under-test-template.yaml
+
+cp ./test-target/cpu-exclusive-pool-rt-sched-policy-pod.yaml /temp/rendered-cpu-exclusive-pool-rt-sched-policy-pod.yaml
+cp ./test-target/cpu-shared-pool-non-rt-sched-policy-pod.yaml /temp/rendered-cpu-shared-pool-non-rt-sched-policy-pod.yaml
```

Comment:
So the real "magic" in the above rendered templates is the "$SCRIPT_DIR"/mo

#/ Mo is a mustache template rendering software written in bash.  It inserts
#/ environment variables into templates.

It'll fill in the {{ }} blocks with variables.

Reply:
`cp ./test-target/cpu-exclusive-pool-rt-sched-policy-pod.yaml /temp/rendered-cpu-exclusive-pool-rt-sched-policy-pod.yaml`

Should be something like:

`cat ./test-target/cpu-exclusive-pool-rt-sched-policy-pod.yaml | "$SCRIPT_DIR"/mo > ./temp/rendered-cpu-exclusive-pool-rt-sched-policy-pod.yaml`